rgw/v2/tests/s3_swift/test_bucket_notifications.py

```diff
@@ -238,6 +240,21 @@ def test_exec(config, ssh_con):
                 if config.test_ops.get("verify_persistence_with_kafka_stop", False):
                     notification.start_stop_kafka_server("stop")
 
+                # Choose the encryption_method sse-s3 or sse-kms
```

Comment:
@hmaheswa Looks these lines are reusable. Please put them in a method in reusable.py and call

Reply:
done @viduship 
please re-review